apps/judicial-system/web/src/routes/Court/components/RulingModifiedModal/JudgeRequestRulingSignatureModal.tsx

```diff
@@ -0,0 +1,25 @@
+import { useIntl } from 'react-intl'
+
+import RequestRulingSignatureModal from './RequestRulingSignatureModal'
+import { strings } from './JudgeRequestRulingSignatureModal.strings'
+
+interface Props {
+  onYes: () => void
+  onNo: () => void
+}
+
+const JudgeRequestRulingSignatureModal: React.FC<
+  React.PropsWithChildren<Props>
```

Comment:
Same as https://github.com/island-is/island.is/pull/14109/files?diff=split&w=1#diff-571b4bfba816052ce935d532394a2e538da701f21bbae75a282784d486cc3bc2R49